armi/reactor/tests/test_components.py

```diff
@@ -962,6 +1080,13 @@ def test_dimensionThermallyExpands(self):
 
 
 class TestHoledHexagon(TestShapedComponent):
+    """Test holed hexagon shaped component.
+
+    .. test:: Holed hexagon shaped component
+        :id: T_ARMI_COMP_SHAPES5
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.